uber/models.py

```diff
@@ -996,7 +991,7 @@ class Attendee(MagModel, TakesPaymentMixin):
     admin_notes = Column(UnicodeText, admin_only=True)
 
     badge_num  = Column(Integer, default=0, nullable=True, admin_only=True)
-    badge_type = Column(Choice(c.BADGE_OPTS), default=c.ATTENDEE_BADGE, admin_only=True)
+    badge_type = Column(Choice(c.BADGE_OPTS), default=c.ATTENDEE_BADGE)
```

Comment:
I forgot that we actually do let the attendee set the badge type in the prereg form, though obviously we check to make sure that someone didn't monkey with the form to give themselves a Guest badge or something.  Therefore this field is not ``admin_only``.